src/helpers/formatAuthorsData.ts

```diff
@@ -0,0 +1,9 @@
+import { mockedAuthorsList } from 'src/constants';
+
+export const formatAuthorsData = (ids: string[]): string => {
+	const authors = mockedAuthorsList.map((author) => {
```

Comment:
You can find a bug on the home page in the Javascript course - `Authors:Vasiliy Dobkin, Nicolas Kim, ,`. This empty strings between two commas in the end are created because `.join` joins all variables of the array (including undefined).

I would use combination of filter, map and join here

```suggestion
	return mockedAuthorsList
	  .filter((author) => {...})
	  .map((author) => {...})
	  .join(', ');
```

Could you finalize the suggestion please?